src/components/TodoItem.tsx

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+/* eslint-disable react/jsx-no-comment-textnodes */
+import React, { useEffect, useRef, useState } from 'react';
+
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todo: Todo;
+  loading: boolean;
```

Comment:
```suggestion
  isLoading: boolean;
```

[Tips on naming boolean variables](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)